app/src/main/java/org/eyeseetea/malariacare/domain/entity/Settings.java

```diff
@@ -3,19 +3,39 @@
 import static org.eyeseetea.malariacare.domain.utils.RequiredChecker.required;
 
 public class Settings {
+
+    public enum MediaListMode  { GRID, LIST }
+    public final MediaListMode DEFAULT_LIST_STYLE = MediaListMode.GRID;
```

Comment:
please not used public fields, the default value should be encapsulated within domain entity, execute SettingsRepository when you need to know the MediaListMode value.